### ISSUE TITLE
Feature to see what's blocked by list/rule

### ISSUE THREAD

**Issue by JS-E** [safing/portmaster#609]
**What would you like to add or change?**:

I would love to see a dashboard or widget to see each application request that has been blocked through my rules or the ad lists. I could click it and see a breakdown of blocks or allows for each application 

**Why do you and others need this?**:

Currently i am using an application that doesn't seem to want to talk to my local server anymore and it's hard to debug why without seeing a trace through or packet tracing - If i could just see-> App tried to talk to server, request dropped due to rule on 'block outgoing on blahblah', request dropped. Also i just tried to access a site and the page didn't load- i thought it was a internet problem but it was portmaster dropping the request because it was on a 'gambling' site list and i only knew because i disabled all the lists for firefox on portmaster.

P.S Love portmaster <3

**Comment by northys** [comment]
@JS-E maybe I don't understand what exactly you are looking for but isn't connection history exactly what you need? on the right above the connection list there is dropdown where you can change the view from by Domain to connections so the connections are chronologicaly ordered.

Just open Portmaster Dashboard/UI whatever you call it and click on the App you are trying to debug. You will see connection history and after you click on any connection there is reason why it got blocked. There are many possible reasons why something got blocked - by portmaster itself, remote dns, your own rule...

**Comment by itsnotsaved** [comment]
>I would love to see a dashboard or widget to see **each application request that has been blocked through my rules or the ad lists**.

I think @JS-E means about something like this. e.g. go to a specific app (e.g. chrome) then click on (1) or (2) and UI display lists of blocked domains* ~/ IP / URLs and unblock needed domain without needing to inspect connection history.

![image](https://user-images.githubusercontent.com/73447325/169152339-83aeb077-f8af-4737-a05f-62fa7762ec3b.png)

**Comment by JS-E** [comment]
^^ Yeah like this above as @itsnotsaved is suggesting - Sorry i wasn't very clear when i explained this

**Comment by dhaavi** [comment]
That a nice idea!

Pinging @ppacher for consideration in the new Network Monitor.
TL;DR: Make the stats tiles add a filter to the search for what they are showing.

**Comment by ppacher** [comment]
Since the new network monitor has been released to the [Beta Release Channel](https://docs.safing.io/portmaster/settings#core/releaseChannel), could you check if grouping by "Allowed" now solves your use-case as well?

**Comment by JS-E** [comment]
Yeah so i think with the new network monitor that has been added really helps debugging problems with blocks on apps so i'm happy that this solves my issue. Nice work! Looks awesome
